### DISCUSSION TITLE
SD Card issues

### DISCUSSION THREAD

**Discussion by philspitlerSF** [forkineye/ESPixelStick#814] [General]
I'm trying to use my EsPixelStick v4.05 to play an FSEQ from the SD card.

I formated the SD card as FAT32 but I still don't see the file management option in the UI.

I tried an 8GB card and 64GB.

Any ideas?

**Comment by MartinMueller2003** [comment]
Use the latest version on the web flasher

**Reply by philspitlerSF** [reply]
I have the firmware installed already so I'm not sure how this will help. Does the web installer have a different version?

**Comment by MartinMueller2003** [comment]
4.05 is about 15 revs back.  cant fix bugs in that version. web flasher has the most recent relases sorted by date

**Reply by philspitlerSF** [reply]
The web installer fixed all my issues... thanks

**Comment by philspitlerSF** [comment]
Amazing. I didn't see the web flasher. I'll look around. 

Thanks
